### ISSUE TITLE
missing exports condition

### ISSUE THREAD

**Issue by ruizdiazever** [metonym/svelte-pincode#22]
https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition

Warning:

1:00:58 PM [vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-pincode@2.2.0

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.

**Comment by metonym** [comment]
Fixed in [v2.2.1](https://github.com/metonym/svelte-pincode/releases/tag/v2.2.1)

**Comment by jneuendorf-i4h** [comment]
@metonym I had some issues using the unstyled inputs. The fix for me was to change the imports like this:

```diff
<script>
-  import Pincode from "svelte-pincode/src/unstyled/Pincode.svelte";
-  import PincodeInput from "svelte-pincode/src/unstyled/PincodeInput.svelte";
+  import Pincode from "svelte-pincode/unstyled/Pincode.svelte";
+  import PincodeInput from "svelte-pincode/unstyled/PincodeInput.svelte";
</script>
```

This removal of `src/` in the path corresponds to the export maps from #23 . So I guess, the documentation should be adjusted, shouldn't it?

**Comment by metonym** [comment]
@jneuendorf-i4h Thank you for flagging. I think the unstyled components are incorrectly mapped in the `exports` map. Will get a fix out.

```
"./unstyled/*.svelte": {
  "types": "./src/unstyled/*.svelte.d.ts",
  "import": "./src/*.svelte"
}
```

**Comment by metonym** [comment]
@jneuendorf-i4h The import mapping for `unstyled` is fixed in [v2.2.2](https://github.com/metonym/svelte-pincode/releases/tag/v2.2.2). I also updated the documentation (the `src/` prefix is no longer needed).

```html
<script>
  import Pincode from "svelte-pincode/unstyled/Pincode.svelte";
  import PincodeInput from "svelte-pincode/unstyled/PincodeInput.svelte";
</script>
```
